src/www/pages/styles/utility-classes/index.njk

```diff
@@ -194,6 +210,21 @@ Where <code class="wmnds-website-inline-code">{size}</code> is one of:
 </pre>
 
 
+<hr>
+{# Background #}
+<h2>Background Colour</h2>
+<p>
+  Changes background to the intended colour.
+</p>
 
+<pre class="wmnds-col-1 wmnds-col-md-1-2">
+  <code class="scss" tabindex="0">
```

Comment:
```suggestion
  <code class="wmnds-show-more-ignore scss" tabindex="0">
```

Adding this class stops the "show more code" bottom from appearing.

I've also noticed this is happening on the "containers" code preview on this page. Can you also add the same class to the code preview too.